build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/biomeTasks.ts

```diff
@@ -40,13 +29,20 @@ export class BiomeTask extends LeafWithFileStatDoneFileTask {
 	 * apply to the directory. Files ignored by git are excluded.
 	 */
 	protected async getInputFiles(): Promise<string[]> {
-		const biomeConfig = await this.biomeConfig;
+		const biomeConfig = await BiomeConfigReader.create(
+			this.node.pkg.directory,
+			this.context.gitRepo,
+		);
```

Comment:
Don't want to keep this cached? Use a helper function that checks for undefined member?